earth_enterprise/src/fusion/gecombineterrain/combineterrainprofiler.cpp

```diff
@@ -0,0 +1,67 @@
+// Copyright 2018 the Open GEE Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <iomanip>
+#include <sstream>
+#include <string>
+#include <sys/types.h>
+#include <unistd.h>
+#include <sys/syscall.h>
+#include <time.h>
+
+#include "common/notify.h"
+#include "combineterrain.h"
+#include "combineterrainprofiler.h"
+
+using namespace std;
+
+// Initialize static members
+CombineTerrainProfiler * const CombineTerrainProfiler::_instance =
```

Comment:
this might not be needed if we change the 'instance()' method implementation as suggested below.

Reply:
See comment below.